package_managers/apt_get/apt_get.bzl

```diff
@@ -14,60 +14,115 @@
 
 """ Rules that create an output script to install packages via apt-get."""
 
-CACHE_DIR = "/install"
+CACHE_DIR = "/tmp/install"
 
 ARCHIVE_DIR = "."
 
+load(
+    "//package_managers:package_manager_provider.bzl",
```

Comment:
Did you forget to check in package_manager_provider.bzl?

Reply:
yes. :)